From 13229528db9e69983bbf10ea5748936f837e53a2 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 10 Nov 2023 00:01:16 -0800
Subject: [PATCH] Fix path to find aes.h

This makes sure it can build when Sourcetee and buildtree are not same

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 drivers/net/wireless/rtl8821cu/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rtl8821cu/Makefile b/drivers/net/wireless/rtl8821cu/Makefile
index c13e87d42660..350f77022d86 100644
--- a/drivers/net/wireless/rtl8821cu/Makefile
+++ b/drivers/net/wireless/rtl8821cu/Makefile
@@ -2456,7 +2456,7 @@ ifeq ($(CONFIG_SDIO_HCI), y)
 rtk_core += core/rtw_sdio.o
 endif
 
-EXTRA_CFLAGS += -I$(src)/core/crypto
+EXTRA_CFLAGS += -I$(src)/core/crypto -I$(srctree)/$(src)/core/crypto
 rtk_core += \
 		core/crypto/aes-internal.o \
 		core/crypto/aes-internal-enc.o \
-- 
2.42.1

